src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/Extensions/ElementNodeOperationExtensions.cs

```diff
@@ -114,7 +115,7 @@ public static void AddSecurityTag(this ElementNode node, ProcessResult result)
             ElementNode newMetaNode = ElementNode.FromElement(meta.ToTypedElement());
             if (metaNode == null)
             {
-                node.Add(s_provider, newMetaNode);
+                node.Add(s_provider, newMetaNode, metaNodeName);
```

Comment:
If we use poco serialization, will it add `meta` or `Meta`?

Reply:
poco serialization will add "meta"

Reply:
And ElementNode serialization will be `Meta`?

Reply:
Yes. Since there is a mapping for Poco between Meta type node and "meta" field Name.

Reply:
Thanks! The variable name looks as a local parameter. Let's either capitalize it or make it starts with "_".

Reply:
Thanks, fixed it.